src/components/ProjectList/ProjectListRow/index.jsx

```diff
@@ -1,34 +1,62 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+
 import mapTime from '../../../mappers/mapTime';
-import { useQuery } from 'urql';
-import { LABELS_QUERY as query } from '../Queries/projectQueries';
 import LetterGradeContainer from '../CodeClimate/LetterGradeContainer.jsx';
-import { title, rtd, rtc } from './projectListRow.module.scss';
-import { labelPreviewDesign } from '../../Settings/Settings.module.scss';
+import LabelDropdown from '../LabelDropdown/index';
+
+import { title, rtd, statusColumns } from './projectListRow.module.scss';
 
-const ProjectRow = ({ project }) => {
-  return (
-    <tr>
-      <td>
-        <div></div>
-      </td>
-      <td className={title}>
-        <Link to={`/project/${project.id}`} className={title}>
-          {project.name}
-        </Link>
-        <LetterGradeContainer
-          ccrepos={project.product.grades}
-        />
-      </td>
-      <td className={rtd}>
-        {project.notes.length
-          ? mapTime(project.notes[0].updatedAt)
-          : mapTime(project.updatedAt)}{' '}
-        ago
-      </td>
-    </tr>
-  );
+const ProjectRow = ({ project, status }) => {
+  const statusLabelsArr = [];
+  var i;
+  if (status.length > 0) {
+    for (i = 0; i < 4 && !(i >= status.length); i++) {
+      statusLabelsArr.push(status[i]);
+    }
+    return (
+      <tr>
+        <td className={title}>
+          <Link to={`/project/${project.id}`} className={title}>
+            {project.name}
+          </Link>
+          <LetterGradeContainer ccrepos={project.product.grades} />
+        </td>
+        {/* <td className={rtd}>
```

Comment:
If you're leaving this code in for a reason please leave a comment explaining why 👍

Reply:
Will do